src/app/tests/crud/test_crud_robot.py

```diff
@@ -0,0 +1,66 @@
+import pytest
+from sqlalchemy.exc import IntegrityError
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from app.models.robot_type import RobotType
+from app.models.robot import Robot
+from app.crud.crud_robot import robot
+from app.schemas.robot import RobotUpdate
+from app.tests.utils.models import create_robot, create_robot_type
+
+pytestmark = pytest.mark.asyncio
+
+
+async def test_crud_robot_create_fail(database: AsyncSession):
+    robot_type_obj = RobotType(id=1, name="Test")
+    try:
+        await create_robot(db=database, robot_type_obj=robot_type_obj)
+        assert False
+    except IntegrityError:
+        assert True
+
+
+async def test_crud_robot_create(database: AsyncSession):
+    robot_type_obj = await create_robot_type(db=database)
+
+    robot_obj = await create_robot(db=database, robot_type_obj=robot_type_obj)
+    assert robot_obj is not None
+    assert isinstance(robot_obj, Robot)
```

Comment:
Please also assert the correctness of the `robot_type_id` of the `robot_obj`.

And: Please also check the that the object has arrived in the database. (And not just that I return the request body and still get a pass :wink:)

Reply:
Addressed